### PR TITLE
commit in dev

### PR DESCRIPTION
**Pull request by Rahul-I2E**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Repo-GitHub-Desktop
  
 
-change from local in main branch
\ No newline at end of file
+change from local in main branch
```
Comment (ANIKETNN) [inline README.md:4]: Looks Fine

Comment (ANIKETNN) [reply README.md:4]: HI

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Repo-GitHub-Desktop
```
Comment (ANIKETNN) [inline README.md:2]: Hi

**Review by ANIKETNN — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rahul-I2E) [reply README.md:4]: The change s are done

File: README.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
  
 
-change from local in main branch
\ No newline at end of file
+change from local in main branch
+
+change from DEV branch
```
Comment (kunwarkundan-I2econsulting) [inline README.md:6]: Update the change details for this pull request.

**Review by kunwarkundan-I2econsulting — CHANGES_REQUESTED** [review]

**Pull request merged**